charts/testkube-logs/values.yaml

```diff
@@ -0,0 +1,60 @@
+# Default values for testkube-logs.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 3
+
+image:
```

Comment:
if we're going to attach this chart to the `testkube` one, then we need to introduce `global` variable here as well: https://github.com/kubeshop/helm-charts/blob/develop/charts/testkube/values.yaml#L7

for example - If we set `global.imageRegistry` then this value will apply to all images in the charts (api, dashboard, operator etc). You may find the examples on how to add global images in the existing charts: https://github.com/kubeshop/helm-charts/blob/develop/charts/testkube-api/templates/deployment.yaml#L334

Reply:
And we also add comments for each parameter in values.yaml, like here for example: https://github.com/kubeshop/helm-charts/blob/develop/charts/testkube-dashboard/values.yaml